Makefile

```diff
@@ -54,7 +54,7 @@ test:
 .PHONY: e2e
 e2e: $(BIN)
 	@echo "--- e2e ---"
-	@E2E_FUNC_E_PATH=$(PWD)/$(BIN) go test -parallel 1 -v -failfast ./e2e
```

Comment:
changing this allowed one less translation problem in cygwin. It is now allowed to be and defaults to relative to the project root.